### ISSUE TITLE
define parameter buffer and dataset get parsed as errors

### ISSUE THREAD

**Issue by gustason** [BalticAmadeus/OpenedgeAblFormatter#246]
For example,
``` abl
define PARAMETER BUFFER  p_Buf  for _Field.
```
![image](https://github.com/user-attachments/assets/e53ec6e5-ec9c-48e2-9cb6-d4d129826212)
And 
``` abl
define INPUT  PARAMETER DATASET FOR dsAudPolicy.
```
![image](https://github.com/user-attachments/assets/99880174-8a36-4305-b9dc-d4afedc0d798)

**Comment by eglekaz** [comment]
Registered: https://github.com/usagi-coffee/tree-sitter-abl/issues/98

**Comment by eglekaz** [comment]
Fixed with version 0.0.44 of tree-sitter-abl
